tests/e2e/utils/setup-analytics.js

```diff
@@ -3,24 +3,31 @@
  */
 import { wpApiFetch } from './wp-api-fetch';
 
-const defaultSettings = {
+const defaultAnalyticsSettings = {
 	accountID: 100,
 	propertyID: 200,
 	profileID: 300,
 	internalWebPropertyID: 400,
 	useSnippet: true,
 };
 
+const defaultAnalytics4Settings = {
+	propertyID: '500',
+	webDataStreamID: '500',
```

Comment:
Maybe let's use `600` as the web data stream ID to not match the propertyID property?
```suggestion
	webDataStreamID: '600',
```

Reply:
Good spot, thanks. I've updated it to `'600'`.